common/lib/capa/capa/responsetypes.py

```diff
@@ -343,9 +344,13 @@ def make_hint_div(self, hint_node, correct, student_answer, question_tag,
         # Tricky: label None means output defaults, while '' means output empty label
         if label is None:
             if correct:
-                label = _(u'Correct')
+                correctness = _("Correct")
             else:
-                label = _(u'Incorrect')
+                correctness = _("Incorrect")
+
+            label = HTML(u'{correctness}').format(
```

Comment:
Now that this is just plain text, it isn't correct to wrap it with HTML. Just go back to label as it was originally.